src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_TYPES = 5;
+    public static final int MAX_RANDOM_SIDE_LENGTH = 5;

```

Comment:
```suggestion
    private static final int NUMBER_OF_FIGURE_TYPES = 5;
    private static final int MAX_RANDOM_SIDE_LENGTH = 5;

```

It's better to make this constants private if we use them only in this class